web/app/js/components/MetricsTable.jsx

```diff
@@ -35,7 +35,7 @@ const formatTitle = (title, tooltipText) => {
   }
 
 };
-const columnDefinitions = (sortable = true, resource, namespaces, onFilterClick, linkifyNsColumn, ConduitLink) => {
+const columnDefinitions = (sortable = true, resource, namespaces, onFilterClick, showNsColumn, ConduitLink) => {
```

Comment:
It feels a little weird to start passing these booleans in. My preferred pattern is to do filtering and/or make it explicit.

```
      <MetricsTable
        resource={friendlyTitle}
        metrics={processedMetrics}
        extraColumns={[NamespaceColumn]} />
```

For a single column, I'm not sure it is worth the work to do this kind of a refactor here ... just something to think about as a pattern.